src/main/java/org/humancellatlas/ingest/config/RepositoryConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package org.humancellatlas.ingest.config;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.humancellatlas.ingest.biomaterial.web.BiomaterialEventHandler;
+import org.humancellatlas.ingest.core.service.ValidationStateChangeService;
+import org.humancellatlas.ingest.file.web.FileEventHandler;
+import org.humancellatlas.ingest.process.ProcessEventHandler;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@RequiredArgsConstructor
```

Comment:
Are you sure we need this constructor?

Reply:
yes, it throws an error on the `ValidationStateChangeService` if I dont use that annotation